xbmc/epg/GUIEPGGridContainer.cpp

```diff
@@ -114,11 +114,11 @@ CGUIEPGGridContainer::CGUIEPGGridContainer(const CGUIEPGGridContainer &other)
   m_focusedProgrammeLayouts = other.m_focusedProgrammeLayouts;
   m_programmeLayouts        = other.m_programmeLayouts;
   m_rulerLayouts            = other.m_rulerLayouts;
-  m_channelLayout           = other.m_channelLayout;
-  m_focusedChannelLayout    = other.m_focusedChannelLayout;
-  m_programmeLayout         = other.m_programmeLayout;
-  m_focusedProgrammeLayout  = other.m_focusedProgrammeLayout;
-  m_rulerLayout             = other.m_rulerLayout;
+  m_channelLayout           = NULL;
+  m_focusedChannelLayout    = NULL;
+  m_programmeLayout         = NULL;
+  m_focusedProgrammeLayout  = NULL;
+  m_rulerLayout             = NULL;
```

Comment:
commit message says it's not called but will likely blow up? if it's not called we might as well = delete the copy constructor